dev-lang/nasm/patches/nasm-2.14.03rc2.patchset

```diff
@@ -0,0 +1,292 @@
+From 17e8687ba790d3bde86556210f41ee02ede37a15 Mon Sep 17 00:00:00 2001
+From: trungnt2910 <trungnt282910@gmail.com>
+Date: Tue, 21 Jan 2020 20:55:56 +0700
+Subject: Fixed various build errors in gcc
+
+Since code in .hpkg packages are only meant to be used for Haiku, I've removed numerous lines of code intended to ensure portablity, but makes Haiku's version of gcc complain.
+
+diff --git a/asm/directiv.h b/asm/directiv.h
+index 20a6424..cf6af11 100644
+--- a/asm/directiv.h
++++ b/asm/directiv.h
+@@ -55,6 +55,9 @@ enum directive {
+ extern const struct perfect_hash directive_hash;
+ extern const char * const directive_tbl[38];
+ 
++/* Prototype for function to avoid build errors */
++const char * invalid_enum_str(int);
++
+ static inline enum directive directive_find(const char *str)
+ {
+     return perfhash_find(&directive_hash, str);
+diff --git a/doc/Makefile b/doc/Makefile
+index 69b8d8b..02cca08 100644
+--- a/doc/Makefile
++++ b/doc/Makefile
+@@ -5,14 +5,14 @@
+ top_srcdir	= ..
+ srcdir		= .
+ 
+-prefix		= /usr/local
++prefix		= /packages/nasm-2.14.03rc2-1/.self
+ exec_prefix	= ${prefix}
+-bindir		= ${exec_prefix}/bin
+-mandir		= ${datarootdir}/man
+-docdir		= ${datarootdir}/doc/${PACKAGE}
++bindir		= /packages/nasm-2.14.03rc2-1/.self/bin
++mandir		= /packages/nasm-2.14.03rc2-1/.self/documentation/man
++docdir		= /packages/nasm-2.14.03rc2-1/.self/documentation/packages/nasm
+ htmldir		= ${docdir}
+-infodir		= ${datarootdir}/info
+-datarootdir	= ${prefix}/share
++infodir		= /packages/nasm-2.14.03rc2-1/.self/documentation/info
++datarootdir	= /packages/nasm-2.14.03rc2-1/.self/data
```

Comment:
Shouldn't be needed, runConfigure should take care of this

Reply:
I forgot to remove that file from my commits. Will remove it soon.